bee-common/bee-common/src/logger/config.rs

```diff
@@ -91,6 +98,28 @@ pub struct LoggerOutputConfig {
     pub(crate) target_exclusions: Vec<String>,
 }
 
+impl LoggerOutputConfig {
+    /// Returns the name of the output file, or `stdout` for standard output.
+    pub fn name(&self) -> &String {
+        &self.name
+    }
+
+    /// Returns the log level of the output.
+    pub fn level_filter(&self) -> LevelFilter {
+        self.level_filter
+    }
+
+    /// Returns the target filters of the output.
+    pub fn target_filters(&self) -> &Vec<String> {
```

Comment:
Can you use `&[String]` to avoid the pointer indirection or is something else using a specific method of `Vec`?

Reply:
Will do 👍